components/Logo/Logo.module.scss

```diff
@@ -0,0 +1,27 @@
+.logoSmall {
+  @extend .logo;
```

Comment:
We also can use **&.classname** inside of parent class, its like a "@extend" syntax sugar. But we can skip that for now